Tests/svgLib/path/parser_test.py

```diff
@@ -226,6 +226,21 @@
                 ("closePath", ()),
             ]
         ),
+        # absolute A command
+        (
+            "M 100 100 A 150 150 0 1 0 150 -150 z",
+            [
+                ('moveTo', ((100.0, 100.0),)),
+                ('qCurveTo', ((100.0, 100.0),
+                    (217.1758387643403, 139.78681395054122),
```

Comment:
Should we round this floats?

Reply:
we are asserting equality, so we can't round.
if you want to round, you could try using `pytest.approx`
https://docs.pytest.org/en/latest/builtin.html#pytest.approx

Reply:
looking at the CI test failures, it looks like we do need to test for almost equality.
Can you try changing line 250 of parser_test.py like this:

```diff
-    assert pen.value == expected
+    assert pen.value == pytest.approx(expected)
```